homes_guests_loves.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Title</title>
+    <link href="homes_guests_loves_styles.css" rel="stylesheet">
+</head>
+<body>
+<section class="homes-guests">
+    <div class="homes-guests-container">
+<h1 class="col-lg-12 col-sm-8">Homes guests loves</h1>
+
+    <div class="home-guests-gallery gallery col-lg-10 col-sm-12">
+
+        <div class="home-guests-box box">
+        <img class="home-guests-photo photo" src="images/homes_images/hotel_leopold.svg" alt="hotel_leopold">
+        <p class="home-hotel-name home-text">Hotel Leopold</p>
+        <p class="home-destination home-text">Saint Petersburg, Russia <span class="hidden">destination</span></p>
+        </div>
+        <div class="home-guests-box box">
+        <img class="home-guests-photo photo" src="images/homes_images/apartment_sunshine.svg" alt="apartment_sunshine">
+        <p class="home-hotel-name home-text">Apartment Sunshine</p>
+        <p class="home-destination home-text">Santa Cruz de Tenerife, Spain</p>
+        </div>
+            <div class="home-guests-box box">
+        <img class="home-guests-photo photo" src="images/homes_images/villa_kunerad.svg" alt="villa_kunerad">
+        <p class="home-hotel-name home-text">Villa Kunerad</p>
+        <p class="home-destination home-text">Vysokie Tatry, Slowakia <span class="hidden">destination</span></p>
+            </div>
+                <div class="home-guests-box box">
+        <img class="home-guests-photo photo" src="images/homes_images/hostel_friendship.svg" alt="hostel_friendship">
+        <p class="home-hotel-name home-text">Hostel Friendship</p>
+        <p class="home-destination home-text">Berlin, Germany <span class="hidden">destination</span></p>
+                </div>
+        <div class="home-switcher">
+            <img class="home-switcher-left" src="images/homes_images/arrow.svg" alt="arrow" width="40" height="40">
+            <img class="home-switcher-right" src="images/homes_images/arrow.svg" alt="arrow" width="40" height="40">
+        </div>
+
+    </div>
+    </div>
+
+</section>
+
+<section class="best-destinations">
+    <div class="best-destinations-container">
+    <h1 class="col-lg-12 col-sm-8">The best destinations</h1>
+    <form>
+            <div class="best-destinations-btns">
+        <button class="best-destinations-btn-regions btn">Regions</button>
+        <button class="best-destinations-btn-cities btn">Cities</button>
+                <button class="best-destinations-btn-places btn">Places <span class="hidden-word">of interest</span></button>
+            </div>
+    </form>
+
+    <div class="best-destinations-gallery gallery col-lg-10 col-sm-12">
+
+        <div class="best-destinations-box box">
+            <img class="best-destinations-photo photo" src="images/destinations_images/Maldives.svg" alt="Maldives">
```

Comment:
content images shouldn't be .svg. It's an image, a photograph, not an icon or logo. Content images should be images. Better use .jpg format, in future you'll just accept the format which user will load